### PR TITLE
CGO45 - Popup Information Bug 

### LINKED ISSUE #45: Highlighting Coordinates and Pop Up Information Missing Buildings
# Description

* There seems to be missing buildings that are highlighted on the map (X, Z, GA)

## Expected Result
* X, Z, GA are highlighted on the map and can see their popup information

## Actual Result
* X, Z, GA are not highlighted 

## Steps
1. Open Application
2. Look for the X, Z, GA buildings on the map
3. They are not on the map

### PR DESCRIPTION
**Pull request by DBatistaFerreira**
#45

* Added Department and Services to Concordia Constants
* Fixes Buildings without popup information

### CODE REVIEW

**Review by danielwiktorczyk — CHANGES_REQUESTED** [review]
Missing UI

**Comment by MoreWaqar** [comment]
Has the requested change been handled? Not sure whether to follow through with review at current time @danielwiktorczyk @Pwnagebird

**Comment by danielwiktorczyk** [comment]
@MoreWaqar we are waiting on the UI component before moving forward with this ticket.

**Review by MoreWaqar — APPROVED** [review]
Approved. Tested the main functionality and everything looks good. Code seems a lot cleaner too.

I think the UI change request is now met and merge can be done.

File: lib/utilities/application_constants.dart [hunk 1]
```diff
@@ -7,3 +7,6 @@ const Color concordiaRed = Color(0xff800206);
 
 const double listElementTextSize = 24.0;
 const double initialedIconSize = 42.0;
+
+double screenHeight;
```
Comment (danielwiktorczyk) [inline lib/utilities/application_constants.dart:11]: Great!

**Review by danielwiktorczyk — CHANGES_REQUESTED** [review]
Excellent UI addition! 
One small thing: bottom left "More". Can it be made into a similar style as the "Go" bottom right icon, maybe with a '?' info icon? 
image

**Review by danielwiktorczyk — APPROVED** [review]
Bottom left UI uses an icon now rather than a textual label, very good decision! 

This ticket looks ready to go to me 😎

image

**Pull request merged**